src/lib.rs

```diff
@@ -53,7 +53,7 @@ mod client;
 pub use client::Client;
 mod endpoints;
 pub use endpoints::{
-    accounts::{Account, Owner},
+    accounts::{Account, Owner, Type as AccountType},
```

Comment:
I would prefer to make the `accounts` module pub so that `Type` is still namespaced under `accounts`. Similarly to the way `feed_items` is exposed.